src/main/java/CountElementsString.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+
+public class CountElementsString {
+    public static void main(String[] args) {
+
+        Scanner reader = new Scanner(System.in);
+        System.out.print("Input any symbol: ");
+        char c = reader.next().charAt(0);
```

Comment:
Would be better to name variables according to it's meaning: at least `char symbol`.

Reply:
done